app/src/main/java/com/framgia/fdms/screen/producer/vendor/VendorPresenter.java

```diff
@@ -1,5 +1,7 @@
 package com.framgia.fdms.screen.producer.vendor;
 
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports